dbt/include/duckdb/macros/adapters.sql

```diff
@@ -1,7 +1,21 @@
-
 {% macro duckdb__create_schema(relation) -%}
   {%- call statement('create_schema') -%}
-    create schema if not exists {{ relation.without_identifier() }}
+    {% set sql %}
+        select type from duckdb_databases()
```

Comment:
Nit-ish question: when was this method added? I want to know if we need to do a bump from the minimal duckdb >= 0.7.0 dependency we include in setup.py

Reply:
Looks good

![image](https://github.com/duckdb/dbt-duckdb/assets/24326299/53a13edd-d8bd-490e-8ac7-ea27a93cdae5)